zeppelin-web-angular/src/app/pages/workspace/notebook/paragraph/code-editor/code-editor.component.ts

```diff
@@ -59,6 +59,7 @@ export class NotebookParagraphCodeEditorComponent implements OnChanges, OnDestro
   @Output() readonly toggleEditorShow = new EventEmitter<void>();
   private editor?: IStandaloneCodeEditor;
   private monacoDisposables: IDisposable[] = [];
+  private highlightDecorations: string[] = [];
```

Comment:
How about defining a type that describes the meaning of the string somewhere and use it for here?
This way, other developers can more quickly understand how this property is indended to be used.
e.g.,
```typescript
type DecorationIdentifier = ReturnType<monaco.editor.ICodeEditor['deltaDecorations']>[number];

@Component({
...
})
export class ...
...
  private highlightDecorations: DecorationIdentifier[] = [];
...
```